BlueEastCode/bluerain-ui-interfaces
# Added Checkbox interface

## What I did
Interface of Checkbox was already there. I modified it to have generic props after observing the three themes

## How to test

Is this testable with jest or storyshots?
No.

Are any of the existing tests are failing ?
You can just observe it.

Does this need an update to the documentation?

Does it have breaking changes ?

If your answer is yes to any of these, please make sure to include it in your PR.


@bjavaid @amnajunaid please review the changes and merge them.
